packages/atomic/src/pages/examples/ipx.html

```diff
@@ -9,9 +9,17 @@
     <script nomodule src="/build/atomic.js"></script>
     <link rel="stylesheet" href="/themes/coveo.css" />
     <script>
-      (async () => {
-        await customElements.whenDefined('atomic-search-interface');
-        const searchInterface = document.querySelector('atomic-search-interface');
+      const ipxSelectorToButtonMode = (searchInterface, hostDiv) => {
+        searchInterface.style.display = 'inline';
+        hostDiv.style.display = 'none';
+      };
+
+      const ipxButtonToSelectorMode = (searchInterface, hostDiv) => {
+        searchInterface.style.display = 'none';
```

Comment:
I'm wondering if that should be in the component itself 🤔 If we scale the IPX, we might start having lots of these small changes that when added up won't make a lot of sense and will be difficult for user to understand and use

Reply:
This exists solely to toggle between the 2 ipx, but yes I agree that if the examples get more complex, we should split them up into separate pages !

Reply:
But if you create an IPX that uses a target selector, you have to always set the style of the `atomic-search-interface` display to `inline` no?